From 6681625907a7ea5d61ea72c20e0bda4b8844bc63 Mon Sep 17 00:00:00 2001
From: Priyajit Ghosh <prghosh@habana.ai>
Date: Wed, 20 Nov 2024 05:40:20 +0200
Subject: [PATCH] [FSW-11114] fs1: add export functions

Functions which are identified to reuse in other
modules like ppboot, preboot are added to
boot_rom.txt file. Boot ROM make file will extract
this symbols from bootrom.elf to export.elf.
export.elf will be used if any other module need
to call the function which part of exports, to
reuse same code from bootrom.

Change-Id: I2faf493b141748d1033515964f9220411d5b7c41
Signed-off-by: Priyajit Ghosh <prghosh@habana.ai>
---

diff --git a/arch/fs1/arc/boot_rom.txt b/arch/fs1/arc/boot_rom.txt
index 81056c6..853431b 100644
--- a/arch/fs1/arc/boot_rom.txt
+++ b/arch/fs1/arc/boot_rom.txt
@@ -1 +1,11 @@
 # put list of symbols, one per line, to be exported
+boot_desc_get_img_desc
+boot_rom_mem_init
+boot_rom_mem_copy
+boot_rom_fw_mem_arc_addr
+boot_rom_spi_dma
+secure_boot_verify_image_hash384
+lfs_file_copy
+lfs_file_set
+lfs_file_copy_partial
+lfs_disk_set_empty
